sdcm/sct_events/base.py

```diff
@@ -38,21 +38,13 @@
 from sdcm import sct_abs_path
 from sdcm.sct_events import Severity, SctEventProtocol
 from sdcm.sct_events.events_processes import EventsProcessesRegistry
+from sdcm.utils.common import Singleton
 
 DEFAULT_SEVERITIES = sct_abs_path("defaults/severities.yaml")
 
 LOGGER = logging.getLogger(__name__)
 
 
-class Singleton(type):
```

Comment:
Moving it out to sdcm.utils.common brings tons of unnecessary dependencies, also common.py is already at 2k lines ,which is not good.
Let's put it into separate file sdcm.utils.singleton.py , I don't like idea of having a file with 8 lines in it, but in this particular case it worth it.

Reply:
I moved to a new module in `sdcm/utils/metaclasses`